textract/layout/analysis.py

```diff
@@ -0,0 +1,240 @@
+import cv2, argparse, os
+import numpy as np
+from pythonRLSA import rlsa
+from collections import defaultdict
+from textract.layout.utils import LayoutType, Page, Paragraph, Image, Line, BBox
+from textract.model.classifier import BlockClassifier, ClassifierType
+
+class Graph(object):
+	def __init__(self, vertex_num):
+		# adjacency List
```

Comment:
looks like the indent is a `Tab`? Highly recommend to use `4 spaces` instead of Tab!